src/cloud/components/Views/Kanban/index.tsx

```diff
@@ -277,10 +277,9 @@ const KanbanView = ({
           </Button>
         </Flexbox>
       </Flexbox>
-      <div className='view--kanban__wrapper'>
```

Comment:
Do not remove classes, they are good to keep for structural integrity and basic debugging

Reply:
Fixed